lib/actions/help.coffee

```diff
@@ -33,7 +33,7 @@ parse = (object) ->
 		return [
 			signature
 			item.description
-		]
+		]).sort()
```

Comment:
This `sort()` means that the primary and secondary commands, listed by `help` and `help -v` respectively, are now listed in alphabetical order (by command name). Previously there didn't seem to be any particular ordering... perhaps the arbitrary order of iteration of properties in a JS object (?).